.github/workflows/release.yml

```diff
@@ -42,19 +42,19 @@ jobs:
         # https://github.com/bazelbuild/bazel/issues/14232#issuecomment-1011247429
         os: ['macos-10.15', 'windows-2019', 'ubuntu-18.04']
         py-version: ['3.7', '3.8', '3.9', '3.10']
-        tf-version: ['2.6.3', '2.8.0']
+        tf-version: ['2.7.1', '2.9.0rc0']
         cpu: ['x86']
         exclude:
           - py-version: '3.10'
-            tf-version: '2.6.3'
+            tf-version: '2.7.1'
         include:
           - os: 'macos-11'
```

Comment:
These are not released yet for 2.9.x https://pypi.org/project/tensorflow-macos/#files

Reply:
Yes, will leave this in the PR though because we don't release until tensorflow-macos has:
https://github.com/tensorflow/addons/issues/2631

Reply:
Still waiting for this

Reply:
@bhack released! :D